packages/wdio-cli/package.json

```diff
@@ -61,7 +60,6 @@
     "lodash.union": "^4.6.0",
     "read-pkg-up": "^10.0.0",
     "recursive-readdir": "^2.2.3",
-    "tsx": "^4.7.2",
```

Comment:
Let's keep this dependency as we use `tsx` in the CLI to transform TypeScript files. The implementation is not working 100% correct but I am working on a patch to improve that.

Reply:
Sure, have updated accordingly.